src/frontend-pwa/src/App.tsx

```diff
@@ -1,55 +1,37 @@
 /**
  * @summary The root component of the application
- * @author Dallas Richmond
- * @author Tyler Maloney
+ * @author Dallas Richmond & Tyler Maloney
  */
 /* eslint-disable no-console */
 import { useState, useEffect } from 'react';
-import axios from 'axios';
 import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import SplashScreen from './components/SplashScreen/SplashScreen';
 import Eula from './views/Eula/Eula';
 import ViewRouter from './routes/ViewRouter';
-import LocationsArray from './Type/LocationsArray';
-import constants from './constants/Constants';
+import useAppService from './services/app/useAppService';
 
 function App() {
-  const [isLoading, setIsLoading] = useState(true);
   const [eulaAccepted, setEulaAccepted] = useState(import.meta.env.DEV || false);
-  const [serviceBCLocations, setServiceBCLocations] = useState<LocationsArray>([]);
-  const [serviceBCServices, setServiceBCServices] = useState<Array<string>>([]);
+  const { state, setLoading } = useAppService();
 
   useEffect(() => {
-    const getData = async () => {
-      try {
-        const { data } = await axios.get(`${constants.BACKEND_URL}/api/locations`);
-        setServiceBCLocations(data.serviceBCLocations);
-        setServiceBCServices(data.serviceBCServices);
-      } catch (error) {
-        console.error(error);
-      }
-    };
-    getData();
-    // TODO: Load in initial data in useEffect, will handle setIsLoading as well.
-    // will be handled in ticket this sprint, specifically [WAYF-101]
-    setIsLoading(false);
+    setLoading(true);
```

Comment:
Since initial state is true, I don't think you need to setLoading here

Reply:
@BradyMitch Yeah you're right! thanks for catching that :)